launchfile_sample.launch

```diff
@@ -0,0 +1,35 @@
+<!--
+	SAMPLE LAUNCH FILE FOR depthimage_to_laserscan for ROS melodic
+	As of September 2nd 2021.
+-->
+
+
+<launch>
+
+    <!-- Depth Image to Laser Scan Node -->
+	<node  name="depthimage_to_laserscan" pkg="depthimage_to_laserscan" type="depthimage_to_laserscan" args="">
```

Comment:
The indentation (here and below) is inconsistent.  Let's use a consistent 2 spaces for indentation for all levels of the XML.